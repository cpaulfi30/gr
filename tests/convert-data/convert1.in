1 g
10 g
100 g
1000 g
10000 g
0.001 kg
0.01 kg
0.1 kg
1 kg
10 kg
100 kg
1000 kg
0.1 lb
1 lb
2 lb
3 lb
0.1 oz
1 oz
10 oz
100 oz
1 ml
10 ml
100 ml
1000 ml
10000 ml
0.001 dl
0.01 dl
0.1 dl
1 dl
10 dl
100 dl
0.001 l
0.01 l
0.1 l
1 l
10 l
