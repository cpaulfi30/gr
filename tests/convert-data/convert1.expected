INPUT '1 g'
RESULT '1 g'

INPUT '10 g'
RESULT '10 g'

INPUT '100 g'
RESULT '100 g'

INPUT '1000 g'
RESULT '1 kg'

INPUT '10000 g'
RESULT '10 kg'

INPUT '0.001 kg'
RESULT '1 g'

INPUT '0.01 kg'
RESULT '10 g'

INPUT '0.1 kg'
RESULT '100 g'

INPUT '1 kg'
RESULT '1 kg'

INPUT '10 kg'
RESULT '10 kg'

INPUT '100 kg'
RESULT '100 kg'

INPUT '1000 kg'
RESULT '1000 kg'

INPUT '0.1 lb'
RESULT '0.1 lb'

INPUT '1 lb'
RESULT '1 lb'

INPUT '2 lb'
RESULT '2 lb'

INPUT '3 lb'
RESULT '3 lb'

INPUT '0.1 oz'
RESULT '0.1 oz'

INPUT '1 oz'
RESULT '1 oz'

INPUT '10 oz'
RESULT '10 oz'

INPUT '100 oz'
RESULT '6.25 lb'

INPUT '1 ml'
RESULT '1 ml'

INPUT '10 ml'
RESULT '10 ml'

INPUT '100 ml'
RESULT '1 dl'

INPUT '1000 ml'
RESULT '1 l'

INPUT '10000 ml'
RESULT '10 l'

INPUT '0.001 dl'
RESULT '0.1 ml'

INPUT '0.01 dl'
RESULT '1 ml'

INPUT '0.1 dl'
RESULT '10 ml'

INPUT '1 dl'
RESULT '1 dl'

INPUT '10 dl'
RESULT '1 l'

INPUT '100 dl'
RESULT '10 l'

INPUT '0.001 l'
RESULT '1 ml'

INPUT '0.01 l'
RESULT '10 ml'

INPUT '0.1 l'
RESULT '1 dl'

INPUT '1 l'
RESULT '1 l'

INPUT '10 l'
RESULT '10 l'

